### PR TITLE
Rename packages.  Refactor instructions in `README`.

### PR DESCRIPTION
**Pull request by kabilar**
- Moved installation instructions to [datajoint-elements/install.md](https://github.com/datajoint/datajoint-elements/blob/main/install.md).
- Moved instructions on running the workflow to the notebooks.
- Rename `element-data-loader` to `element-interface`
- Version lock requirements

### CODE REVIEW

**Review by CBroz1 — APPROVED** [review]

**Comment by kabilar** [comment]
Added
- Rename `element-animal` to `element-subject`
- Rename `element-data-loader` to `element-interface`

**Comment by kabilar** [comment]
Reverted package rename.  Will keep the package name `element-animal` instead of `element-subject`.

File: README.md [hunk 1]
```diff
@@ -1,282 +1,51 @@
-# Pipeline for extracellular electrophysiology using Neuropixels probe and kilosort clustering method
+# DataJoint Workflow - Array Electrophysiology
 
-Build a full ephys pipeline using the canonical pipeline elements
+Workflow for extracellular array electrophysiology data acquired with a 
+Neuropixels probe using the `SpikeGLX` or `OpenEphys` software and processed 
+with MATLAB- or python-based `Kilosort`.
```
Comment (dimitri-yatsenko) [inline README.md:5]:
```suggestion
Workflow for extracellular array electrophysiology data acquired with a polytrode probe (e.g. 
Neuropixels, Neuralynx) using the `SpikeGLX` or `OpenEphys` acquisition software and processed 
with MATLAB- or python-based `Kilosort` spike sorting software.
```

**Review by dimitri-yatsenko — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kabilar) [reply README.md]: Thanks!

**Review by dimitri-yatsenko — APPROVED** [review]

**Pull request merged**